### ISSUE TITLE
Add Prompt After Searching

### ISSUE THREAD

**Issue by zo-shin** [segler-alex/RadioDroid#399]
When users search a non-existent station name, the app doesn’t give any prompt, so they don’t know if they should keep waiting for the result. A prompt like **no result found** is required after a non-existent item is searched.

**Comment by jo99jo** [comment]
Yes, as a new user, this is one of the first problems I noticed.

**Comment by Wikinaut** [comment]
related to https://github.com/segler-alex/RadioDroid/issues/309
